packages/edit-site/src/components/pattern-actions/rename-menu-item.js

```diff
@@ -19,29 +19,48 @@ import { decodeEntities } from '@wordpress/html-entities';
 /**
  * Internal dependencies
  */
-import { TEMPLATE_POST_TYPE } from '../../utils/constants';
+import {
+	TEMPLATE_POST_TYPE,
+	TEMPLATE_PART_POST_TYPE,
+	PATTERN_TYPES,
+	TEMPLATE_ORIGINS,
+	POST_TYPE_LABELS,
+} from '../../utils/constants';
 
-export default function RenameMenuItem( { template, onClose } ) {
-	const title = decodeEntities( template.title.rendered );
-	const [ editedTitle, setEditedTitle ] = useState( title );
+export default function RenameMenuItem( { postType, postId, onClose } ) {
```

Comment:
While we're trying to make things more consistent, I noticed this gets passed the type and id separately, but in other places, we're passing an item object. 

It might be good to tweak that so all these related components are consumed in an intuitive manner. Would it also prevent any excessive re-renders, not passing a fresh object around?

What do you think?

Reply:
This makes sense. Thanks for flagging. I'll take a look.